day_two_guide/topics/increasing_docker_storage.adoc

```diff
@@ -103,12 +103,57 @@ volumes associated with container storage:
 +
 ----
 # docker-storage-setup
+----
++
+.. For *thin pool* setups, you should see the following output and can proceed to the next step:
++
+----
 INFO: Volume group backing root filesystem could not be determined
 INFO: Device /dev/xvdb is already partitioned and is part of volume group docker_vol
 INFO: Device node /dev/xvdd1 exists.
   Physical volume "/dev/xvdd1" successfully created.
   Volume group "docker_vol" successfully extended
 ----
++
+.. For *XFS* setups that use the Overlay2 file system, the above increase will not be visible.
++
+* Run the `lvextend` command to grow the logical volume to use all the available space in the volume group:
```

Comment:
Do these 3 steps need to be run in order? If so, should they be numbered?

Reply:
all the -> all of the

Reply:
> Do these 3 steps need to be run in order? If so, should they be numbered?

Yes, good point. I've numbered them.